fusillade/resource.py

```diff
@@ -153,6 +187,18 @@ def list_policies(self, per_page=None, next=None):
         children, next_token = self.cd.list_object_children_paged(f"{self.object_ref}/policy", next, per_page)
         return [f"/resource/{self.name}/policy/{child}" for child in children.keys()], next_token
 
+    def list_ids(self, per_page=None, next=None):
+        children, next_token = self.cd.list_object_children_paged(f"{self.object_ref}/id", next, per_page)
```

Comment:
Instead of passing a paging token around, it may be worthwhile to convert `list_object_children_paged` into a generator yielding the objects in `result['Children']` with transparent paging.

Reply:
`self.cd.list_object_children` would return an iterator, but in this case  this function will later be used in the Swagger API and will need paging support.